helm-charts/charts/kube-starrocks/Chart.yaml

```diff
@@ -42,4 +44,4 @@ keywords:
 sources:
   - https://github.com/StarRocks/starrocks
 
-home: https://github.com/StarRocks/starrocks-kubernetes-operator
+home: https://github.com/StarRocks/starrocks-kubernetes-operator
```

Comment:
make sure your IDE append a new line to the end of the file. avoid change this EOL back and forth.